### ISSUE TITLE
Cat vs Dog

### ISSUE THREAD

**Issue by Charishma-Bailapudi** [pranjay-poddar/Dev-Geeks#1165]
**Describe the Feature you want to end**
I wanna apply the same normalization to X_val and X_test as done for X by dividing the pixel values by 255.
By the way I want to adjust the test size in train_test_split to allocate a smaller portion of the data for testing. For example, to use 20% of the data for testing, you can set test_size=0.2.
As well as I want to experiment with different architectures or modify the existing architecture to improve the model's performance.I can add more layers, change the number of filters, kernel sizes, etc., to explore different configurations.

**Expected behavior**
It will give better  accuracy in classifying the dog and cat.

**Screenshots (optional)**

**Additional context (optional )**

**Comment by khadesamrudhi** [comment]
please assign me this issue under GSSoC'23.
